Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -545,10 +588,11 @@ def audit_bitbucket_repository
       new_formula_problem warning
     end
 
-    def get_repo_data(regex)
+    def get_repo_data(regex, new_formula_only = true)
       return unless @core_tap
       return unless @online
-      return unless @new_formula
+
+      return unless @new_formula || !new_formula_only
```

Comment:
```suggestion
```

And instead change the existing callers to only call this method if `@new_formula` is set.